ui/components/app/gas-details-item/gas-details-item.js

```diff
@@ -50,6 +50,16 @@ const GasDetailsItem = ({ userAcknowledgedGasMissing = false }) => {
     return null;
   }
 
+  const maxPriorityFeePerGasToRender = (
+    maxPriorityFeePerGas ??
+    hexWEIToDecGWEI(transactionData?.txParams?.maxPriorityFeePerGas ?? '0x0')
+  ).toString();
+
+  const maxFeePerGasToRender = (
+    maxFeePerGas ??
+    hexWEIToDecGWEI(transactionData?.txParams?.maxFeePerGas ?? '0x0')
+  ).toString();
```

Comment:
Nit: The `?` after `transactionData` seems unnecessary; it's guaranteed to at least be an empty object (see the `useDraftTransactionGasValues` hook)


```suggestion
  const maxPriorityFeePerGasToRender = (
    maxPriorityFeePerGas ??
    hexWEIToDecGWEI(transactionData.txParams?.maxPriorityFeePerGas ?? '0x0')
  ).toString();

  const maxFeePerGasToRender = (
    maxFeePerGas ??
    hexWEIToDecGWEI(transactionData.txParams?.maxFeePerGas ?? '0x0')
  ).toString();
```